app/assets/javascripts/libs/toast.js

```diff
@@ -30,10 +30,12 @@ const Toast = {
       toastMessage = message;
     }
 
+    const timeOutSeconds = parseInt(timeout / 1000);
```

Comment:
Why `parseInt`? Does ant not accept floats? If you want to round, I'd suggest to use `Math.round` since it's more idiomatic (`parseInt` is for parsing string encoded integer).

Reply:
I used it because of *brainfart* lol